frontend/src/components/lastBuildElement/LastBuild.js

```diff
@@ -6,7 +6,7 @@ import BuildInfoTable from './BuildInfoTable';
 import { useParams } from 'react-router';
 import { useStateValue } from '../../contexts/state';
 
-const Divi = styled.div`
+const TableHeading = styled.div`
```

Comment:
Correct me if I'm wrong, but isn't this supposed to be `caption` instead of `div`?